plugins/modules/activation_key.py

```diff
@@ -49,6 +49,7 @@
     description:
       - List of subscriptions that include either Name, Pool ID, or Upstream Pool ID.
       - Pool IDs are preferred since Names and Upstream Pool IDs are not guaranteed to be unique. The module will fail if it finds more than one match.
+      - NOTE: New versions no longer support this parameter
```

Comment:
The YAML gods don't like that. This is now an entry that is itself a dict `{'NOTE': 'New versions…'}`.
You need to either quote this as a whole string:
```suggestion
      - "NOTE: New versions no longer support this parameter"
```
Or rephrase to not have a colon:
```suggestion
      - This parameter is not supported in SCA mode.
```
(I prefer that, as it also removes the ambiguity what "new versions" are.)

Reply:
I wasn't sure what the proper upstream subscription language was so I avoided adding SCA.